docs/design/mono/mobile-runtimeconfig-json.md

```diff
@@ -0,0 +1,143 @@
+# Mobile runtimeconfig.json host configuration
+
+## Motivation
+
+`runtimeconfig.json` is a file used by .NET 5+ applications to pass arbitrary configuration parameters via the application host to the .NET runtime.  On mobile and WebAssembly, we would like to support passing configuration properties from the Mono embedder to the runtime.
+
+To minimize the impact of the app startup time, the design should preserve these contraints:
+1. Don’t parse JSON file at runtime at application startup
+2. Don’t need to support full set of well-known properties, just the ones that make sense on mobile
```

Comment:
I think you can get rid of this, we support everything

Reply:
Removed item 2